Octokit.Reactive/Clients/IObservableRepositoriesClient.cs

```diff
@@ -23,6 +23,15 @@ public interface IObservableRepositoriesClient
         /// <returns>An <see cref="IObservable{Repository}"/> instance for the created repository</returns>
         IObservable<Repository> Create(string organizationLogin, NewRepository newRepository);
 
+        /// <summary>
+        /// Creates a new repository using a repository template.
+        /// </summary>
+        /// <param name="templateOwner">The owner of the template</param>
+        /// <param name="templateRepo">The name of the template</param>
+        /// <param name="newRepository"></param>
```

Comment:
:memo: please document this new parameter

Reply:
Documentation added